src/subscription_manager/cli_command/service_level.py

```diff
@@ -92,6 +92,13 @@ def _validate_options(self):
                 "can be used only on unregistered systems"
             ))
 
+        if self.is_registered() and (
```

Comment:
The method `self.is_registered()` is called on line 84. Why not extend previous block of code and  error message?

Reply:
I wanted to keep the message separated, because it is shared with the rest of the syspurpose subcommands. The `serverurl` argument is only available for service-level, I kind of wanted to make it an extra. I see the pros and cons in both of the options; I'll update it to match your suggestion.